hugegraph-server/hugegraph-dist/pom.xml

```diff
@@ -293,22 +293,6 @@
                                                 <include name="${final.name}/**"/>
                                             </tarfileset>
                                         </tar>
-                                        <!-- copy the artifacts to root
-                                             directory for better user experience -->
-                                        <!-- REMOVE ME after revision 1.5.0 -->
-                                        <exec executable="cp"
-                                              dir="${project.basedir}"
-                                              failonerror="false">
-                                            <arg value="-r"/>
-                                            <arg value="../${final.name}"/>
-                                            <arg value="../../${final.name}"/>
-                                        </exec>
-                                        <exec executable="cp"
-                                              dir="${project.basedir}"
-                                              failonerror="false">
-                                            <arg value="../${final.name}.tar.gz"/>
-                                            <arg value="../../${final.name}.tar.gz"/>
-                                        </exec>
```

Comment:
due to we already have the release script in `dist` module, we don't need cp the pkg now